docs/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Introduction
+sidebar_position: 1
+---
+
+import ArchitecturalModel from '../src/components/ArchitecturalModel'
+
+## About
+
+The Sovereign Cloud Stack combines the best of Cloud Computing in one unified standard.
+SCS is built, backed, and operated by an active open-source community worldwide.
+
+## Architectual Overview
+
+<ArchitecturalModel jsonFilePath="data/architecturalLayerData.json" />
+
+## Use Cases and Deployment Examples
+
+### IaaS Layer
+
+#### Quick Start with Cloud-In-A-Box
+
+The fastest way to get in touch with SCS is to deploy a SCS cloud virtually. The Cloud-In-A-Box was built explicitly for this scenario. Check it out [here](/docs/iaas/guides/deploy-guide/examples/cloud-in-a-box)
+
+#### Reference Implementation Testbed
+
+This means that you set up an SCS test installation including all the infrastructure
+pieces such as database, message queueing, ceph, monitoring and logging, IAM, the
+[OpenStack](https://openstack.org/) core services, and (soon) the Container layer
+on top of an existing IaaS platform.
+
+The SCS IaaS reference implementation is based on [OSISM](https://osism.tech/). Read on the
+[OSISM testbed docs](https://docs.osism.de/testbed/) to learn how to get the
+testbed running. Please read carefully through the
+[deployment](https://docs.osism.de/testbed/deployment.html) section of the
+manual.
+
+### Container Layer
+
+#### k8s Cluster API Provider
+
+You can easily deploy the container layer on top of the testbed (or a production
+SCS cloud of course) checking out the code from
+[k8s-cluster-api-provider](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/).
+
+#### Cluster Stacks
+
+With Cluster Stacks, the V2 KaaS reference implementation, we provide an opinionated optimized configuration of Kubernetes clusters. Through better packaging, integrated testing, and bundled configuration SCS-based Kubernetes clusters can be individualized much easier.  
+Throughout the R6 development cycle Cluster Stacks are taken from a technical preview to be [functional and available on top of the IaaS reference implementation](https://github.com/SovereignCloudStack/issues/milestone/8) as well to replace the V1 KaaS reference implementation [k8s-cluster-api-provider](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/).  
```

Comment:
```
Throughout the R6 development cycle Cluster Stacks are taken from a technical preview to be [functional and available on top of the IaaS reference implementation](https://github.com/SovereignCloudStack/issues/milestone/8) as well to replace the V1 KaaS reference implementation [k8s-cluster-api-provider](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/).  
```

@chess-knight Could you reword this? I believe that cluster stacks will be used as part of the `k8s-cluster-api-provider`, not instead or as a replacement.

Reply:
AFAIK `k8s-cluster-api-provider` is **KaaS v1** reference implementation and `cluster-stacks` is **KaaS v2** so it can be called a *replacement*.